index.html

```diff
@@ -551,31 +563,41 @@ <h3>
         </h3>{{SerialPort/onconnect}} is an [=event handler IDL attribute=] for
         the {{connect}} event type.
         <p>
-          When a serial port becomes available on the system that the user has
-          allowed the site to access as the result of a previous call to
-          {{Serial/requestPort()}}, run the following steps:
+          When a serial port that the user has allowed the site to access as the
+          result of a previous call to {{Serial/requestPort()}} becomes
+          [=logically connected=], run the following steps:
         </p>
         <ol>
           <li>Let |port:SerialPort| be a {{SerialPort}} representing the port.
           </li>
+          <li>Set |port|.{{SerialPort/[[connected]]}} to `true`.
+          </li>
           <li>[=Fire an event=] named {{connect}} at |port| with its
           {{Event/bubbles}} attribute initialized to `true`.
           </li>
         </ol>
+        <p>
+          A serial port is <dfn>logically connected</dfn> if it is a wired
+          serial port and the port is physically connected to the system, or if
+          it is a wireless serial port and the wireless device is registered
+          with the system.
```

Comment:
"Registered with the system" sounds like "paired" to me, but the problem we need to solve in [crbug/1488031](https://bugs.chromium.org/p/chromium/issues/detail?id=1488031) is a device that is registered with the system (paired) but has had all of its open connections intentionally closed in order to logically disconnect it.

I would phrase this here as:

> [...] it is a wireless serial port and the system has active connections to the wireless device (e.g. an open Bluetooth L2CAP channel).

In the `requestPort()` and `getPort()` steps we want to return devices even if `connected` is `false` as calling `port.open()` would establish that connection and in some circumstances that is desirable behavior. For example if the user actively clicked a "Connect" button.

Reply:
Thanks, I've updated it so we have separate definitions for "logically connected" and "available". "Available" ports are included in `requestPort`/`getPort` while "logically connected" is used for connection events and the `connected` attribute.